proposal/jptom/README.rst

```diff
@@ -0,0 +1,99 @@
+===========================
+Proposal for Kalman filter 
+===========================
+
+Basic information
+=================
+
+`My project repository <https://github.com/jptom/study-probablistic-robotics/tree/main/nsd-term-project>`_
+
+My base python code is from this book.
+This author translated `Probabilistic Robotics <https://www.amazon.com/Probabilistic-Robotics-INTELLIGENT-ROBOTICS-AUTONOMOUS-ebook/dp/B00DJD9LXC>`_ in 2005.
+This book introduce it's implementation in python by him in 2019.
+
+.. image:: https://images-na.ssl-images-amazon.com/images/I/91C20m7ZgoL.jpg
+   :scale: 20%
+   
+Problem to solve
+================
+
+
+.. image:: image/kalmansim.PNG
+.. image:: image/kalman.PNG
+
+Kalman filter is a technique of self-localazation of robotics.
```

Comment:
Spelling: localization.  There are other spelling errors. It will be good if you can use a spelling check for the document. I highlight this out since it appears twice.